webpack.config.js

```diff
@@ -0,0 +1,38 @@
+var path = require('path');
```

Comment:
@ukilon-okta I think that'll be part of phase II - first get webpack, and then add babel so we can start using import and other es6+ functions